app/controllers/loans_controller.rb

```diff
@@ -12,31 +12,48 @@ def new
       @user = User.find_by(id: params[:user_id])
       unless @user.good_to_borrow?
         flash[:error] = "User Can Not Borrow at This Time"
-        redirect_to user_path(@user.id) and return 
-      end 
+        redirect_to user_path(@user.id) and return
+      end
       @loan = Loan.new(user: @user)
     elsif params[:book_id]
       @book = Book.find_by(id: params[:book_id])
-      unless @book.available 
+      unless @book.available
         flash[:error] = "Book is Not Available"
-        redirect_to book_path(@book.id) and return 
-      end 
+        redirect_to book_path(@book.id) and return
+      end
       @loan = Loan.new(book: @book)
```

Comment:
Maybe we can clear the book's selected status so when they go back to selected books they can see what's left.

@book.selected = false